zeppelin-web/src/app/jobmanager/job/job.component.test.js

```diff
@@ -0,0 +1,125 @@
+import { ParagraphStatus } from '../../notebook/paragraph/paragraph.status'
+
+describe('JobComponent', () => {
+  const baseUrlSrvMock = { getRestApiBase: () => '' }
+  let $componentController
+  let $httpBackend
+
+  beforeEach(angular.mock.module('zeppelinWebApp'))
+  beforeEach(angular.mock.inject((_$componentController_, _$httpBackend_) => {
+    $componentController = _$componentController_
+    $httpBackend = _$httpBackend_
+  }))
+
+  it('should get progress when there is a finished paragraph', () => {
+    const paragraphs = [
+      { status: ParagraphStatus.FINISHED },
+    ]
+    const mockNote = createMockNote(paragraphs)
+    const bindings = { note: mockNote, }
+
+    const ctrl = $componentController('job', null, bindings)
+    expect(ctrl).toBeDefined()
+
+    const progress1 = ctrl.getProgress()
+    expect(progress1).toBe('100%')
+  })
+
+  it('should get progress when there is pending and finished paragraphs', () => {
+    const paragraphs = [
+      { status: ParagraphStatus.PENDING },
+      { status: ParagraphStatus.FINISHED},
+    ]
+    const mockNote = createMockNote(paragraphs)
+    const bindings = { note: mockNote, }
+
+    const ctrl = $componentController('job', null, bindings)
+
+    const progress1 = ctrl.getProgress()
+    expect(progress1).toBe('50%')
+  })
+
+  it('should get proper job type icons', () => {
```

Comment:
test descriptions (`should get proper job type icons `) are duplicated in this one and the next one